File: Python/sorts/bubble_sort.py
import random
import datetime

x = []

start = datetime.datetime.now()

for i in range(0, 100):
    numberToAppend = int(round(random.random() * 10000))
    x.append(numberToAppend)

def bubbleSort(list):
    unsorted = True
    length = len(list) - 1
    while unsorted:
        unsorted = False
        for idx in range(0, length):
            if list[idx] > list[idx + 1]:
                temp = list[idx]
                list[idx] = list[idx + 1]
                list[idx + 1] = temp
                unsorted = True
        length -= 1
    totalTimeElapsedInSeconds = (datetime.datetime.now() - start).total_seconds()
    print "Total time elapsed in seconds:", totalTimeElapsedInSeconds
    return list

print bubbleSort(x)

File: Javascript/Sorts/bubble_sort.js
var arr = [11, 8, 9, 5, 3];

function bubbleSort(arr) {
    var k = arr.length;
    var swapping = true;
    while(swapping) {
        swapping = false;
        for (var i = 0; i < k - 1; i++) {
            if (arr[i] > arr[i + 1]) {
                var temp = arr[i];
                arr[i] = arr[i + 1];
                arr[i + 1] = temp;
                swapping = true;
            }
        }
        k--;
    }
    return arr;
}

var newArr = bubbleSort(arr);
console.log(newArr);